### DISCUSSION TITLE
Quantity Amounts

### DISCUSSION THREAD

**Discussion by Dilosen** [ghostfolio/ghostfolio#3557] [Ideas]
Hi There, 

Currently, quantity denomination is set to two decimal places, thus 0.00 - In many instances, especially with fractional shares, this ends up being several decimal places; for example, $50 in BTC is equivalent to 0,00086 at the time of this post.  In the activities section of GF, this will show as 0.00.

Is there a way to increase this so that we can see the value of quantity?  

I currently have a quantity mismatch, and thus account mismatch, and I am unable to view the quantites for each activity to isolate any import mistakes.

**Comment by Dilosen** [comment]
Bumping this post - any help would be appreciated.
I don't want to have to go into the codebase to change it, in case of breaking changes. 

My chart for Bitcoin is looking flat because it shows 0.00 and not the BTC denominations

**Reply by dtslvr** [reply]
> My chart for Bitcoin is looking flat because it shows 0.00 and not the BTC denominations

Have you checked the _Bitcoin_ holding of the demo account (see https://ghostfol.io/en/demo)? Your chart should look like this and not flat:

<img width="772" alt="btc-chart" src="https://github.com/user-attachments/assets/d30bb811-9ab9-4195-b572-12faa79ebcd9">

**Reply by Dilosen** [reply]
Hi 

No the chart looks like this
![awsedfw](https://github.com/user-attachments/assets/3f4e09d5-95e5-483e-b0e7-e9a900c35814)

I have about 35 entries here and all of them are much smaller than 0.00 - this is a problem. The demo is 0.56 so it'll display fine. 

Not sure how we can expose the full denominations in the activities panel?

**Comment by ruudduurruud** [comment]
same here

**Comment by Dilosen** [comment]
I haven't gone into the code yet, I assume that this would break the app if I do it or it'll not be a permenant fix once it gets updated. Hopefully @dtslvr will be able to provide some guidence

**Comment by soloam** [comment]
Try going to your "Admin Controls" menu and then "Market Data" look for the bitcoin asset and click on it. Then click on the top right three vertical dots and run "Gather Historical Data" this should pull all historical data to that asset. 

If this does not work pull the page down and you should see the "Benchmark" box, select it and exit the page (you can do cancel, it's a bug that when you select Benchmark it does not activate the Save button, but you do cancel and it saves it). Go back in into the asset and double check if Benchmark is selected, if yes run "Gather Historical Data" (on the top right three vertical dots) again.

Now you should wait to the job to run... if you would like to speed it up go to "Job Queue" on the "Admin Controls" menu and if there is a job there click the 3 dots and make it run...

See if this solved your issue

**Comment by Dilosen** [comment]
Thanks, running your steps now and will see if it works. Tried to execute the job but it's status is a red clock. So I'll give it some time, see if it works and then try the steps again if it does not. Will update this on the result

**Reply by swissbuechi** [reply]
It can only fetch data which is not older than 1 year with coincecko. You'll need to manually import the prices using the CSV with data provided from Yahoo Finance for example.

**Reply by Dilosen** [reply]
fetching data is not the problem, it's the quantity denominations that are the problem.
Bitcoin is 100 millionth of one BTC, thus the denominations are more than just two decimal spaces. 

simply put, a $50 purchase of BTC shows a quantity of 0.00 instead of 0.001239242 or whatever the denomination. Thus, when you total it'll still show 0.00 for most of my transactions irrespective of price. 

Hence why my chart looks the way it does. 

Bitcoin needs to be more than two decimal spaces
